deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -102,43 +127,83 @@ def run_logic(self, context: Context) -> CheckResult:
+            train_prediction = np.array(model.predict_proba(train_dataset.features_columns))
+            test_prediction = np.array(model.predict_proba(test_dataset.features_columns))
+        else:
+            train_prediction = np.array(model.predict(train_dataset.features_columns)).reshape((-1, 1))
+            test_prediction = np.array(model.predict(test_dataset.features_columns)).reshape((-1, 1))
+
+        samples_per_class = train_dataset.label_col.value_counts().to_dict()
+
+        drift_score_dict = {}
+        drift_display_dict = {}
+        method = ''
+        classes = train_dataset.classes
+
+        for class_idx in range(train_prediction.shape[1]):
+            class_name = classes[class_idx]
+            drift_score_dict[class_name], method, drift_display_dict[class_name] = calc_drift_and_plot(
+                train_column=pd.Series(train_prediction[:, class_idx].flatten()),
+                test_column=pd.Series(test_prediction[:, class_idx].flatten()),
+                value_name='model predictions' if not proba_drift else
+                           f'predicted probabilities for class {class_name}',
+                column_type='categorical' if (train_dataset.label_type != TaskType.REGRESSION) and (not proba_drift)
+                            else 'numerical',
+                margin_quantile_filter=self.margin_quantile_filter,
+                max_num_categories_for_drift=self.max_num_categories_for_drift,
+                max_num_categories_for_display=self.max_num_categories_for_display,
+                show_categories_by=self.show_categories_by,
+                categorical_drift_method=self.categorical_drift_method,
+                with_display=context.with_display,
+            )
 
         if context.with_display:
-            headnote = """<span>
+            headnote = f"""<span>
                 The Drift score is a measure for the difference between two distributions, in this check - the test
-                and train distributions.<br> The check shows the drift score and distributions for the predictions.
+                and train distributions.<br> The check shows the drift score and distributions for the predicted 
+                {'class probabilities' if proba_drift else 'classes'}.
             </span>"""
 
-            displays = [headnote, display]
+            # sort classes by their drift score
+            print(drift_score_dict)
+            displays = [headnote] + [x for _, x in sorted(zip(drift_score_dict.values(), drift_display_dict.values()),
+                                                          reverse=True)][:self.max_classes_to_display]
         else:
             displays = None
 
-        values_dict = {'Drift score': drift_score, 'Method': method}
+        values_dict = {'Drift score': drift_score_dict, 'Method': method, "Samples per class": samples_per_class}
 
         return CheckResult(value=values_dict, display=displays, header='Train Test Prediction Drift')
```

Comment:
If you are displaying multiple plots, could be cool to display them in tabs instead of one after the other.
Easy implementation can be seen in WeakSegmentPerformance

Reply:
Keeping it as is for now so it'll look like the other drift checks. You can open an issue to do that in all drift checks separately.